tests/dbt/test_manifest.py

```diff
@@ -38,6 +42,8 @@ def test_manifest_helper():
     assert waiter_revenue_by_day_config.dependencies == Dependencies(
         macros={
             MacroReference(name="log_value"),
+            MacroReference(name="should_full_refresh"),
```

Comment:
Why has this set been updated?

Reply:
The `is_incremental` macro calls `should_full_refresh`. dbt did not report the dependency, so it wasn't picked-up originally.